test/test_multibroker.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-2016: Alignak team, see AUTHORS.txt file for contributors
+#
+# This file is part of Alignak.
+#
+# Alignak is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Alignak is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Alignak.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+"""
+This file test the multibroker in schedulers
+"""
+
+import requests_mock
+from alignak.macroresolver import MacroResolver
+from alignak.http.scheduler_interface import SchedulerInterface
+from alignak_test import AlignakTest
+
+
+class TestMultibroker(AlignakTest):
+    """
+    This class test the multibroker in schedulers
+    """
+
+    def test_multibroker_onesched(self):
+        """
+        Test with 2 brokers and 1 scheduler
+
```

Comment:
Few explanations will be welcome to explain the expected behavior ... when we will go back to this test, it is not clear what it is doing exactly ...